Tests/AlicerceTests/UIKit/UIImageTestCase.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIImageTestCase.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 03/08/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+import UIKit
+
+@testable import Alicerce
+
+final class UIImageTestCase: XCTestCase {
+
+    func testOriginal_WhenHasImage_ItShouldReturnTheOriginalImage() {
+        let mrMinder = imageFromFile(withBundleClass: UIImageTestCase.self,
+                                     name: "mr-minder",
+                                     type: "png")
+
+        let mrMinderOriginal = mrMinder.original
+
+        XCTAssertEqual(mrMinderOriginal.renderingMode, .alwaysOriginal)
+    }
+
+    func testOriginal_WhenHasImage_ItShouldReturnTheTemplateImage() {
+        let mrMinder = imageFromFile(withBundleClass: UIImageTestCase.self,
+                                     name: "mr-minder",
+                                     type: "png")
+
+        let mrMinderOriginal = mrMinder.template
```

Comment:
I'm guessing this name came along after the copy & paste. I'm guessing it should be `mrMinderTemplate`? :smile:

Reply:
@portellaa the only thing I see that might be missing is this ☝️

Reply:
ups 😂

Reply:
you guys must be crazy, don't know what you are talking about 😝 

btw, fixed 😂